ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/ReadSymbolsFromMobileStyle/ReadSymbolsFromMobileStyle.qml

```diff
@@ -0,0 +1,215 @@
+// [WriteFile Name=ReadSymbolsFromMobileStyle, Category=DisplayInformation]
+// [Legal]
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.11
+import QtQuick.Controls 2.5
+import QtQuick.Layouts 1.3
+import Esri.Samples 1.0
+
+Item {
+
+    // add a mapView component
+    MapView {
+        id: view
+        anchors.fill: parent
+
+        Button {
+            anchors {
+                bottom: view.attributionTop
+                horizontalCenter: parent.horizontalCenter
+                bottomMargin: 10
+            }
+            text: "Clear Graphics"
+            onClicked: model.clearGraphics();
+        }
+    }
+
+    // Declare the C++ instance which creates the scene etc. and supply the view
+    ReadSymbolsFromMobileStyleSample {
+        id: model
+        mapView: view
+    }
+
+    Rectangle {
+        anchors {
+            fill: optionGrid
+            margins: -5
+        }
+        color: "#efefef"
+    }
+
+    GridLayout {
+        id: optionGrid
+        anchors {
+            left: parent.left
+            top: parent.top
+            margins: 10
+        }
+
+        columns: 2
+
+        Label {
+            Layout.columnSpan: 2
+            Layout.alignment: Qt.AlignCenter
+            text: "Change Options to Modify Symbol"
+            font.underline: true
+        }
+
+        Label {
+            text: "Eyes"
+        }
+
+        ComboBox {
+            id: eyeComboBox
+            model: model.eyesResults
+            textRole: "name"
+            delegate: Item {
+                height: 30
+                width: parent.width
+
+                RowLayout {
+                    Image {
+                        source: symbolUrl
+                        Layout.preferredWidth: 20
+                        Layout.preferredHeight: 20
+                    }
+                    Label {
+                        text: name
+                    }
+                }
+
+                MouseArea {
+                    anchors.fill: parent
+                    onClicked: {
+                        eyeComboBox.currentIndex = index;
+                        eyeComboBox.popup.close();
+                    }
+                }
+            }
+            onCurrentTextChanged: updateSymbol()
+        }
```

Comment:
Is there any way to use a Repeater to reduce the duplication of code? It looks like the label and the combobox are both duplicated three times with minor differences.